src/css/footer.css

```diff
@@ -0,0 +1,88 @@
+.footer .cotainer {
+    background-color: #7a3145;
+    padding: 40px 20px;
+}
+.cotainer {
+    max-width: 375px;
+    padding: 80px 20px;
+    margin: 0 auto;
+}
+*, ::after, ::before {
+    box-sizing: border-box;
+}
+div {
+    display: block;
+    unicode-bidi: isolate;
+}
+.footer {
+color: #fbfbfb;
+font-size: 14px;
+font-style: normal;
+font-weight: 500;
+line-height: 1.28;
+}
+.footer-info {
+    margin-bottom: 80px;
+}
+.footer-first-div {
+    margin-bottom: 40px;
+}
+.footer-logo {
+    display: flex;
+    align-items: center;
+    margin-bottom: 14px;
+}
+a {
+    color: currentColor;
+    text-decoration: none;
+}
+p {
+    margin: 0;
```

Comment:
we had already remove all margins and paddings in reset css, there no need to remove it again